src/pystatis/table.py

```diff
@@ -46,13 +58,25 @@ def get_data(self, area: str = "all", prettify: bool = True, **kwargs):
 
         self.raw_data = raw_data_str
         data_buffer = StringIO(raw_data_str)
-        self.data = pd.read_csv(
-            data_buffer, sep=";", na_values=["...", ".", "-", "/", "x"]
-        )
+
+        if params["language"] == "en":
+            self.data = pd.read_csv(
```

Comment:
instead of repeating the whole pd command, set a value for a variable that is used for the decimal, like:

```python
decimal_char = "." if params["language"] == "en" else ","
pd.read_csv(..., decimal=decimal_char)
```